### ISSUE TITLE
ctrl+p file search returns "recent files" from files not within my currently opened project. Can we disable this?

### ISSUE THREAD

**Issue by Zephni** [microsoft/vscode#133373]
This has happened too many times to me now where I am working on multiple projects within the same day, and after opening a new site directory within vscode and then pressing `ctrl+p` I click one of the top files expecting it to be within the directory I am sitting in, but then after making loads of changes and testing I realise nothing is changing... and that's because vscode has presented the recently opened file from the last project I was working on that had the same filename. This happens to me all the time and I wonder if others have this same issue?

In short, is there a way to disable recent files entirely so that I don't get this issue, or even better.. can we add an option to disable files from other workspaces or outside directories from displaying within recent files for the `ctrl+p` menu? If not is this an option that can be added?

Thanks, I would be very interested if anyone else would appreciate this feature :)

**Comment by rdsignr** [comment]
100% agreed, this is very disruptive..

**Comment by rdsignr** [comment]
Yes, I found this since I posted, and it does what I need. Thanks
